database/changesets/20201031_WorkflowOptions.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+                        http://www.liquibase.org/xml/ns/dbchangelog
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+    <changeSet author="jwarren@broadinstitute.org"
+               dbms="postgresql"
+               id="20201031"
+               logicalFilePath="20201031_WorkflowOptions.xml">
+        <comment>
+            Add workflow options attributes to existing workflow types
+        </comment>
+        <sql>
+            ALTER TYPE AllOfUsArrays
+                ADD ATTRIBUTE workflow_options   text  CASCADE   -- workflow options to be passed to Cromwell
+        </sql>
+        <sql>
+            ALTER TYPE copyfile
+                ADD ATTRIBUTE workflow_options   text  CASCADE   -- workflow options to be passed to Cromwell
+        </sql>
+        <sql>
+            ALTER TYPE ExternalWholeGenomeReprocessing
```

Comment:
As above - this type is deprecated and nothing will be stored using it anymore. We need to make sure we can still load it, however, and the only place we can handle this in the loader.
If anything, not updating this table will make the load previous wgs test more accurate.

Reply:
Done--this is removed from liquibase